docs/guides/missionmaking.md

```diff
@@ -40,6 +40,11 @@ Przygotowując misję, będziesz chciał (a w naszej grupie, nawet musiał) korz
 
 Aby ustawić modlistę na misji, wystarczy że wybierzesz mody (lub skorzystasz z naszych gotowych modlist) i uruchomisz grę. Przy każdym kolejnym uruchomieniu Army pamiętaj żeby uruchamiać tę samą modlistę. W razie pomyłki edytor pokaże czego brakuje.
 
+**UWAGA**
+Przy tworzeniu misji używaj modlisty tylko z modami wymaganymi, bez modów opcjonalnych. Później może to powodować problemy. 
```

Comment:
It would be good to point out that some clientside mods can be used like `3denEnhanced`, `ArmaZeusCache` or `ZEC` composition mods.